src/PantherTestCaseTrait.php

```diff
@@ -163,10 +163,12 @@ protected static function createPantherClient(array $options = [], array $kernel
 
         self::startWebServer($options);
 
+        $browserArguments = \array_key_exists('browser_arguments', $options) && \is_array($options['browser_arguments']) ? $options['browser_arguments'] : [];
```

Comment:
The default value should be null since an empty array would remove all the default arguments.

Reply:
Ahh, ok! Will switch default to `null` again.